packages/techdocs-common/src/stages/generate/helpers.ts

```diff
@@ -133,6 +133,10 @@ export const runCommand = async ({
 
     process.stderr.on('data', stream => {
       logStream.write(stream);
+      // added below change to make OS ( child process) exception to the backend logs
+      const bufferbase64 = Buffer.from(stream, 'base64');
+      const textAscii = bufferbase64.toString('ascii');
+      console.log(' Process Stderror trace: ', textAscii);
```

Comment:
I don't think this is what we want. Console.log is normally not allowed. Where logging is involved, we pass around the winston logger from the backend instead; see other parts of the codebase

Reply:
This was added to debug the local run and it helped to identify that "techdocs-core"  plugins  issue. This looks more of a process/documentation change rather than logger change.  Please advise.

Reply:
Ideally you should something like `techdocs-cli` locally to preview docs when you are writing it. It should let you know about the mkdocs and its techdocs-core plugin issues.
https://backstage.io/docs/features/techdocs/troubleshooting#mkdocs-build-errors
https://github.com/backstage/techdocs-cli